### PR TITLE
[MM-56011] Migrated radio_setting.tsx from Class to Functional Component #25603

### PR DESCRIPTION
**Pull request by ZubairImtiaz3**
#### Summary
Migrated radio_setting.tsx from Class to Functional Component

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/25603
Jira https://mattermost.atlassian.net/browse/MM-56011

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/radio_setting.tsx [hunk 1]
```diff
@@ -15,47 +15,49 @@ interface Props {
...
-                inputId={this.props.id}
-                helpText={this.props.helpText}
-                setByEnv={this.props.setByEnv}
-            >
-                {options}
-            </Setting>
-        );
-    }
-}
+export default RadioSetting;
```
Comment (larkox) [inline webapp/channels/src/components/admin_console/radio_setting.tsx:63]: This used to be a Pure Component, so use `React.memo` here.

**Review by larkox — CHANGES_REQUESTED** [review]
Just one minor change. Otherwise, it looks great :smile:

File: webapp/channels/src/components/admin_console/radio_setting.tsx (hunk 1 above)
Comment (ZubairImtiaz3) [reply webapp/channels/src/components/admin_console/radio_setting.tsx]: @larkox Changes done.

**Comment by harshilsharma63** [comment]
hi @ZubairImtiaz3 can you check the failing tests in CI?

**Comment by ZubairImtiaz3** [comment]
@harshilsharma63 Update snapshots.

**Review by harshilsharma63 — APPROVED** [review]

**Pull request merged**